### PR TITLE
annotation of PXD006195

### PR DESCRIPTION
**Pull request by dirkwink**
PXD006195 contains a SILAC-labeled data set. Since I do not have much experience with SILAC, I would be very grateful for a double check.

### CODE REVIEW

File: annotated-projects/PXD006195/sdrf.tsv [hunk 1]
```diff
@@ -0,0 +1,73 @@
+source name	characteristics[organism]	characteristics[organism part]	characteristics[cell type]	characteristics[disease]	assay name	comment[fraction identifier]	comment[label]	comment[label]	comment[data file]	comment[associated file uri]	comment[instrument]	comment[dissociation method]	comment[cleavage agent details]	comment[fragment mass tolerance]	comment[fragment mass tolerance]	comment[modification parameters]	comment[modification parameters]	comment[modification parameters]	factor value[label]	factor value[temperature]
```
Comment (julianu) [inline annotated-projects/PXD006195/sdrf.tsv:1]: I guess the labels should be
AC=PRIDE:0000611;NT=SILAC light R:12C(6)14N(4)	AC=PRIDE:0000613;NT=SILAC light K:12C(6)14N(2)
here (and then every other line)

Comment (dirkwink) [reply annotated-projects/PXD006195/sdrf.tsv:1]: I suppose your right. I created a fix.

**Review by ypriverol — CHANGES_REQUESTED** [review]
Dear @dirkwink First thanks for your contribution. I was reviewing the annotations and they all look great. However, no major biological metadata has been annotated in the dataset and associated with the `factor value`. It is difficult to know what the authors have been trying to analyze and how this is related with each SILAC channel.

**Comment by dirkwink** [comment]
Thanks for reviewing. I will look into the paper again and submit some more information soon.

**Comment by ypriverol** [comment]
ping @dirkwink

**Comment by dirkwink** [comment]
I guess, I can work it out at the end of the week. Have to finish another work first.

**Comment by dirkwink** [comment]
Hey @ypriverol 
I add some factor values to make sure the intentions of the authors.

**Pull request merged**